CONTRIBUTING.md

```diff
@@ -1,10 +1,16 @@
 # Reporting Issues with xDrip
+Before opening an issue, please connect with us by posting in the support channels:  
```

Comment:
There should be an empty line after every heading and there is no need for double-empty lines in most cases.

"Before you open an issue, please contact us by posting in the support channels:"

Reply:
Why should there be an empty line after each heading?

Reply:
The [GFM spec](https://github.github.com/gfm/#atx-headings) does not require an empty line after a heading bit it increases the readability in plain text mode and is recommend (see [Empty Lines Before And After](https://arcticicestudio.github.io/styleguide-markdown/rules/headings.html#empty-lines-before-and-after). The reason is likely historical because it allows to change between ATX and Settext headings. The later require a blank line because it supports multiline headings. But that's off-topic.

Reply:
I have placed a line after every heading now.